src/components/Header/Header.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import {
+  Flex,
+  Heading,
+  IconButton,
+  Text,
+  useColorModeValue
+} from '@chakra-ui/react';
+import i18next from 'i18next';
+
+import {
+  LogoWalletIcon,
+  NotificationsIcon,
+  SettingsIcon,
+  UserIcon
+} from '../../assets';
```

Comment:
@/assets, i think @lazzzaroni set up a way to import from root

Reply:
fixed